lib/src/flutter_deck_router.dart

```diff
@@ -63,7 +74,7 @@ class FlutterDeckRouter {
               key: state.pageKey,
               restorationId: state.pageKey.value,
               transitionsBuilder: slide.configuration.transition.build,
-              child: slide.widget.build(context),
+              child: _FlutterDeckSlidePage(slide),
```

Comment:
No need for any custom widgets, the simple `Builder` should do the trick:

```suggestion
              child: Builder(builder: slide.widget.build),
```